forge-gui/res/cardsfolder/upcoming/agency_coroner.txt

```diff
@@ -0,0 +1,9 @@
+Name:Agency Coroner
+ManaCost:4 B
+Types:Creature Ogre Cleric
+PT:3/6
+A:AB$ Draw | Cost$ 2 B Sac<1/Creature> | NumCards$ X | SpellDescription$ Draw a card. If the sacrificed creature was suspected, draw two cards instead.
```

Comment:
I think Sac should be Creature.Other

How does the StackDescription work out on this one?

Reply:
![image](https://github.com/Card-Forge/forge/assets/67333662/15c2e202-049d-4514-9e1c-8360cbd3c818)

I think that's fine ?

Reply:
Yep, that's great as long as it also shows "two" when the sacced creature was sus

Reply:
any ideas ?

Reply:
I think it might do it automatically... did you check?

Reply:
I will, though the annoying thing is that there a no cards with suspect on this branch lol

Reply:
Yup it works 
![image](https://github.com/Card-Forge/forge/assets/67333662/824af2f4-50b8-48aa-9010-a27c4f4e09b0)
 !